.github/workflows/ci.yml

```
@@ -70,11 +70,14 @@ jobs:
       - name: Init cache
         uses: Swatinem/rust-cache@v2
 
-      - name: Install Fuel toolchain
-        uses: FuelLabs/action-fuel-toolchain@v0.6.0
+      - name: Install fuel toolchain
+        uses: cachix/install-nix-action@v20
         with:
-          toolchain: latest
-          date: 2023-04-12
+          nix_path: nixpkgs=channel:nixos-unstable
+          extra_nix_config: |
+            substituters = https://fuellabs.cachix.org
+            trusted-public-keys = fuellabs.cachix.org-1:3gOmll82VDbT7EggylzOVJ6dr0jgPVU/KMN6+Kf8qx8=
```

Seems like it worked better before, at least it was pulling from the cache!

```suggestion
            extra-substituters = https://fuellabs.cachix.org
            extra-trusted-public-keys = fuellabs.cachix.org-1:3gOmll82VDbT7EggylzOVJ6dr0jgPVU/KMN6+Kf8qx8=
```

---

Yeah, left a comment [here](https://github.com/kayagokalp/sway-applications-nix-ci/pull/1#discussion_r1223456409) about it.